tools/src/bin/sst-metadata.rs

```diff
@@ -91,6 +123,52 @@ async fn run(args: Args) -> Result<()> {
         metas.push(meta);
     }
 
+    if args.stats {
+        let mut file_stats = FileStatistics::default();
+        let mut field_stats_map = HashMap::new();
+        for (object_meta, sst_metadata, metadata_size, kv_size) in metas {
```

Comment:
It seems those parse is duplicated with before, a better way is to collect those size info here
https://github.com/CeresDB/ceresdb/blob/9a9c0f79116dff6df2a09fc8766e6952fd5f4432/tools/src/bin/sst-metadata.rs#L122

and print those size at last.

Reply:
fixed